int32[12] tactile_value
int32 crc_errors